project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._, Keys._
 object Dependencies {
 
   val ScalaVersions = Seq("2.11.11", "2.12.2")
-  val AkkaVersion = "2.4.17"
+  val AkkaVersion = "2.4.18"
```

Comment:
We need to make sure that Alpakka works with 2.5.x and 2.4.x, therefore leaving the older version here. We could use 2.5.x here if we could run tests with jars from an older series.